### ISSUE TITLE
Placing ship on board

### ISSUE THREAD

**Issue by yuejOSU** [cs361fall2018/project-fortnitemasters#31]
Hoping to be able to place all three ships on the 10x10 with no overlaps, duplicates, or out of range.

**Comment by yuejOSU** [comment]
I have been able to put a ship on the board, but there is some wonky bug that it places two even though I have printed out that the function only goes through it once. It might be a front-end bug??

**Comment by yuejOSU** [comment]
I might have an idea why. I think that the AI is also placing their on the same board instead of the opposite side, and therefore it shows two of the same ships on different or overlapping coordinates.

**Comment by safarovd** [comment]
Yeah, I see this. It's placing the opposite version of the ship on top right?
